pkg/descheduler/evictions/evictions.go

```diff
@@ -318,7 +308,7 @@ func (pe *PodEvictor) Evictable(opts ...func(opts *Options)) *evictable {
 }
 
 // IsEvictable decides when a pod is evictable
-func (ev *evictable) IsEvictable(pod *v1.Pod) bool {
+func (ef *EvictorFilter) Filter(pod *v1.Pod) bool {
```

Comment:
so eventually, as a plugin users will customize this by implementing their own `EvictorFilter.Filter()` function?

Reply:
Yes. `Filter` (alongside `Sort`) is going to be a new extension point.